website/source/docs/provisioning/ansible_local.html.md

```diff
@@ -79,6 +79,9 @@ This section lists the _specific_ options for the Ansible Local provisioner. In
 
     The default value is `:default`, and any invalid value for this option will silently fall back to the default value.
 
+  - `:pip_args`: When Ansible is installed via pip this option allows the defiitional of optional pip arguments to be included (for example,
```

Comment:
More generally, the documentation is not exact. This PR proposes to add a new provisioner option `pip_args`, not an optional argument to the `install_mode` option. Can you please reword this, and also mention its default value, similarly to other option documentations. Thanks ❤️